examples/dt/nova/nova-three-cells/README.md

```diff
@@ -0,0 +1,185 @@
+| Glance           | Swift                   | Must have          |
+| Swift            | (default)               | Must have          |
+| Horizon          | N/A                     | Must have          |
+| Neutron          | Geneve (OVN)            | Must have          |
+| Swift            | (default)               | Must have          |
+
+#### Support services
+
+The following table lists services which are not the main focus of the testing
+(which may be covered by additional scenarios), but are required for the DT
+to work properly and can be deployed with any/default configuration.
+
+| Service          | Reason                     |
+| ---------------- |--------------------------- |
+| Keystone         | needed by all services     |
+
+
+### Additional configuration
+
+- Always-on, default services and features: TLSe
+- Two additional compute nodes
+
+#### Nova
+
+Increase default cells to include cell2
+
+```YAML
+---
+spec:
+  nova:
+    template:
+      secret: osp-secret
+      cellTemplates:
+        cell0:
+          cellDatabaseAccount: nova-cell0
+          cellDatabaseInstance: openstack
+          cellMessageBusInstance: rabbitmq
+          conductorServiceTemplate:
+            replicas: 3
+          hasAPIAccess: true
+        cell1:
+          cellDatabaseAccount: nova-cell1
+          cellDatabaseInstance: openstack-cell1
+          cellMessageBusInstance: rabbitmq-cell1
+          conductorServiceTemplate:
+            replicas: 3
+          hasAPIAccess: true
+        cell2:
+          cellDatabaseAccount: nova-cell2
+          cellDatabaseInstance: openstack-cell2
+          cellMessageBusInstance: rabbitmq-cell2
+          conductorServiceTemplate:
+            replicas: 3
+          hasAPIAccess: true
+```
+
+#### RabbitMQ
+
+Add a new rabbitmq service for cell2
+
+```YAML
+---
+spec:
+  rabbitmq:
+    templates:
+      rabbitmq-cell2:
+        replicas: 3
+```
+
+#### Galera
+
+Create a third galera service for cell2
+
+```YAML
+spec:
+ galera:
+    templates:
+      openstack-cell2:
+        storageClass: lvms-local-storage
+        storageRequest: 5G
+        secret: osp-secret
+        replicas: 3
+```
+
+#### OSDPNS
+
+Create a second nodeset that references a new compute service an necessary
+secret/transport url for cell2
+
+```YAML
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneNodeSet
+metadata:
+  name: openstack-edpm-2
+  namespace: openstack
+spec:
+  services:
+  - bootstrap
+  - download-cache
+  - configure-network
+  - validate-network
+  - install-os
+  - configure-os
+  - ssh-known-hosts
+  - run-os
+  - reboot-os
+  - install-certs
+  - ovn
+  - neutron-metadata
+  - libvirt
+  - nova-cell-2
+
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova-cell-2
+  namespace: openstack
+spec:
+  addCertMounts: false
+  caCerts: combined-ca-bundle
+  containerImageFields:
+  - NovaComputeImage
+  - EdpmIscsidImage
+  dataSources:
```

Comment:
here we should add that ``nova-cell<N>-metadata-neutron-config``, if running metadata service in a cell instead of top scope

Reply:
ack I'll try for per cell deployment.